packages/lib/BaseApplication.ts

```diff
@@ -510,6 +510,10 @@ export default class BaseApplication {
 			}
 		}
 
+		if (action.type === 'NOTE_UPDATE_ONE') {
+			refreshNotes = true;
+		}
```

Comment:
This would refresh the note list every time a note is updated, including while it's being updated so that's definitely not the fix

Reply:
I have been tracking down this issue, and I found another [bug](https://discourse.joplinapp.org/t/editing-note-which-is-selected-from-a-tag-breaks/36830?u=7adidaz) that drove me crazy while debugging on the `dev` branch.. but roman didn't get to reproduce it.. if you have the time... please take a look.

Reply:
I resolved the issue in a new commit, waiting for your feedback.